truffle.js

```diff
@@ -1,91 +1,77 @@
-require('ts-node/register/transpile-only')
+require('ts-node/register/transpile-only');
 
-var HDWalletProvider = require('@truffle/hdwallet-provider')
-var mnemonic = 'digital unknown jealous mother legal hedgehog save glory december universe spread figure custom found six'
+var HDWalletProvider = require('@truffle/hdwallet-provider');
+var mnemonic =
+    'digital unknown jealous mother legal hedgehog save glory december universe spread figure custom found six';
 
-const secretMnemonicFile = './secret_mnemonic'
-const fs = require('fs')
-let secretMnemonic
+const secretMnemonicFile = './secret_mnemonic';
+const fs = require('fs');
+let secretMnemonic;
 if (fs.existsSync(secretMnemonicFile)) {
-  secretMnemonic = fs.readFileSync(secretMnemonicFile, { encoding: 'utf8' })
+    secretMnemonic = fs.readFileSync(secretMnemonicFile, { encoding: 'utf8' });
 }
 
 module.exports = {
-  networks: {
-    coverage: { // coverage/trace provider. note that it currently can't run extrnal-process relay.
-      provider: require('./coverage-prov.js'),
-      verbose: process.env.VERBOSE,
-      network_id: '*'
-    },
-    development: {
-      verbose: process.env.VERBOSE,
-      host: '127.0.0.1',
-      port: 4444,
-      network_id: 33,
-      gas: 6300000,
-      gasPrice: 60000000 // 0.06 gwei
-    },
-    testing: {
-      verbose: process.env.VERBOSE,
-      host: '127.0.0.1',
-      port: 4444,
-      network_id: 33,
-      gas: 6300000,
-      gasPrice: 60000000 // 0.06 gwei
-    },
-    rskdocker: {
-      verbose: process.env.VERBOSE,
-      host: 'enveloping-rskj',
-      port: 4444,
-      network_id: 33,
-      gas: 6300000,
-      gasPrice: 60000000 // 0.06 gwei
-    },
-    rsk: {
-      verbose: process.env.VERBOSE,
-      host: '127.0.0.1',
-      port: 4444,
-      network_id: 33,
-      gas: 6300000,
-      gasPrice: 60000000 // 0.06 gwei
+    networks: {
```

Comment:
Actually the change was made since i was trying to fix the tests workflow, the thing was that we had a lot of non-used networks, so now i just leave the development (that's default on truffle), regtest for regtest use, testnet for testnet use and mainnet for mainnet use, that's all what we need actually. All the other networks were not being used (i think those were on the old GSN code and we don't have that anymore so it's ok to remove it.

Reply:
sounds good, thanks